website_require_login/tests/test_ir_http.py

```diff
@@ -0,0 +1,37 @@
+from odoo.tests import HttpCase
+
+
+class TestIrHttp(HttpCase):
+    def setUp(self):
```

Comment:
can you please move this code to `setUpClass`?

Reply:
Thanks for the input. Done!
Will do the same for #1019